src/services/ls-migration/README.md

```diff
@@ -6,23 +6,23 @@ We migrate the Address Book and Added Safes.
 It can be potentially extended to more pieces of stored data, as each piece is a separate module.
 
 ## How it works
-The new app will be deployed on a new domain, so we have to use and iframe + [postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).
+The new app will be deployed on a new domain, so we have to use an iframe + [postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).
 
 Here's how we do it.
 
 #### On safe-react's side:
 
  * safe-react exposes a special public HTML file named `migrate.html`
  * this file is listening to window messages
- * it ignores anything that is not on the trusted domain list (only web-core domains)
- * if the sender domain is in the trusted list, it sends back the entire contents of the local storage
+ * it ignores anything that is not on a trusted domain list that includes only web-core domains
+ * if the sender domain is in the trusted list, safe-react sends back the entire contents of the local storage
```

Comment:
```suggestion
 * if the sender domain is on the trusted list, safe-react sends back the entire contents of the `localStorage` (LS)
```

Reply:
I'm referring to the storage as a place, not as a JS variable here.

Reply:
Ok, I'd still include the abbreviation in brackets though as it's referenced several times below.